include/Gaffer/TweakPlug.inl

```diff
@@ -179,4 +181,95 @@ bool TweaksPlug::applyTweaks(
 	return tweakApplied;
 }
 
+template<typename T>
+T vectorAwareMin( const T &v1, const T &v2 )
```

Comment:
Seems a bit odd for this to be in a public namespace. Could either put it in a `Detail` sub-namespace, or make it a private static method of `TweakPlug`?

Reply:
Made private.